internal/scorecard/alpha/tests/olm.go

```diff
@@ -70,6 +77,29 @@ func SpecDescriptorsTest(bundle registry.Bundle) scapiv1alpha2.ScorecardTestResu
 	r.State = scapiv1alpha2.PassState
 	r.Errors = make([]string, 0)
 	r.Suggestions = make([]string, 0)
+	csv, err := bundle.ClusterServiceVersion()
+	if err != nil {
+		r.Errors = append(r.Errors, err.Error())
+		r.State = scapiv1alpha2.ErrorState
+		return r
+	}
+	r.Log += fmt.Sprintf("Loaded ClusterServiceVersion: %s\n", csv.GetName())
```

Comment:
@jmccormick2001 is there a reasonable way to either remove `ScorecardTestResult.Log` or make it an `io.Writer` or `strings.Builder{}`? Passing around a potentially massive string as a struct field seems clunky, although it's not a memory hit.

Reply:
the Log field shouldn't be getting much action I'd think, a test just populates it a single time and that struct later gets printed in text format or in json format.  I think the bundle validate test is the only test that makes use of that field if I recall.  That said, it is a free-form string field with no limits on it, we might want to limit the size of that field in the future.